resources/js/Pages/Home.vue

```diff
@@ -1,46 +1,27 @@
 <template>
   <div>
     <Head title="Mismatch Finder" />
-    <h1>Mismatch Finder</h1>
-    <p>This page is coming soon!</p>
-    <p>{{ $i18n('test-string') }}</p>
-    <auth-widget v-bind:user="user" />
-    <Button type="progressive" variant="primary" @click.native="console">Test Wikit Component</Button>
+    <h1 class="h4">{{ $i18n('about-mismatch-finder-title') }}</h1>
+    <p id="about-description" >{{ $i18n('about-mismatch-finder-description') }}</p>
   </div>
```

Comment:
It seems like the "Find out more" link is missing. Other than that it looks amazing and fantastic. :grinning:

Reply:
I didn't include it since so far it is not supposed to do anything, once we implement the dialog that it opens, we should add it :D.

Reply:
Ah, ok. I suspected it might be on purpose but wasn't sure.